src/pages-and-resources/PagesAndResources.jsx

```diff
@@ -36,7 +41,13 @@ function PagesAndResources({ courseId, intl }) {
   return (
     <PagesAndResourcesProvider courseId={courseId}>
       <main className="container container-mw-md">
-        <h3 className="mt-5 mb-5">{intl.formatMessage(messages.heading)}</h3>
+        <div className="d-flex justify-content-between mt-5 mb-5">
+          <h3>{intl.formatMessage(messages.heading)}</h3>
+          <a href={lmsCourseURL}>
```

Comment:
Why can't we have Paragon `Link` here? that opens in a new tab? Is it a requirement to open the link in the same page?

Reply:
that's not mentioned anywhere but I just kept the old experience, it was opening in the same tab previously

Reply:
confirmed need to open in a new tab